integration-tests/olminstall/helpers/create_eaas_cluster.py

```
@@ -0,0 +1,168 @@
+
+
+def main() -> int:
+    instance_type = require_env("INSTANCE_TYPE")
+    version = require_env("VERSION")
+    kubeconfig_value = require_env("KUBECONFIG_VALUE")
+    tenant = require_env("TENANT")
+    result_path = require_env("CLUSTER_NAME_RESULT_PATH")
+    timeout = os.environ.get("TIMEOUT", "30m").strip() or "30m"
+    ics_value = os.environ.get("ICS_VALUE", "").strip()
+
+    kubeconfig = Path("/tmp/kubeconfig")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/kubeconfig"

(S108)